src/components/Modals/NewCourse.vue

```diff
@@ -136,66 +243,106 @@ export default {
           }
         }
       });
-      function addActive(x) {
-        /* a function to classify an item as "active": */
-        if (!x) return;
-        /* start by removing the "active" class on all items: */
-        removeActive(x);
-        if (currentFocus >= x.length) currentFocus = 0;
-        if (currentFocus < 0) currentFocus = x.length - 1;
-        /* add class "autocomplete-active": */
-        x[currentFocus].classList.add('autocomplete-active');
-      }
-      function removeActive(x) {
-        /* a function to remove the "active" class from all autocomplete items: */
-        for (let i = 0; i < x.length; i += 1) {
-          x[i].classList.remove('autocomplete-active');
-        }
-      }
-      function closeAllLists(elmnt) {
-        /* close all autocomplete lists in the document,
-        except the one passed as an argument: */
-        const x = document.getElementsByClassName('autocomplete-items');
-        for (let i = 0; i < x.length; i += 1) {
-          if (elmnt !== x[i] && elmnt !== inp) {
-            x[i].parentNode.removeChild(x[i]);
-          }
-        }
-      }
       /* execute a function when someone clicks in the document: */
       document.addEventListener('click', e => {
         closeAllLists(e.target);
       });
     },
-
     addCourse() {
       if (this.$refs[`dropdown-${this.semesterID}`].value) this.$emit('addItem', this.semesterID);
+    },
+    checkIfLast(elem, list) {
+      return elem === list[list.length - 1];
+    },
+    toggleEditMode() {
+      this.editMode = !this.editMode;
+    },
+    reset() {
+      this.editMode = false;
+      this.selected = false;
+      this.selectedCourse = '';
     }
   }
 };
 </script>
 
 <style lang="scss">
+@import "@/assets/scss/_variables.scss";
 // TODO: font family
```

Comment:
Just because I see this here now, can you please delete this TODO? We didn't have the right font in CoursePlan for a few months, but it is right now 😄